src/commands/redis_cmd.cc

```diff
@@ -1652,7 +1660,10 @@ class CommandHRange : public Commander {
   }
 
  private:
-  int64_t limit_ = LONG_MAX;
+  bool reversed_ = false;
+  int64_t offset_ = 0;
+  int64_t count_ = -1;
+  HashRangeSpec spec_;
```

Comment:
You can correct me if I'm wrong.
Suggestion: you can remove `offset` and `count` since you have them as members of `HashRangeSpec`.
Additionally, to avoid casting, perhaps you can change the type of `count` from `int64_t` to `uint64_t`.

Reply:
Yes, we need negative numbers. There are some redundant variables that can be removed.